eva/parser/parser_visitor/_load_statement.py

```diff
@@ -18,11 +18,28 @@
 from eva.parser.evaql.evaql_parser import evaql_parser
 
 from eva.parser.table_ref import TableRef
+from eva.parser.types import FileFormatType
 
 
 class Load(evaql_parserVisitor):
     def visitLoadStatement(self, ctx: evaql_parser.LoadStatementContext):
         file_path = self.visit(ctx.fileName()).value
         table = TableRef(self.visit(ctx.tableName()))
-        stmt = LoadDataStatement(table, file_path)
+
+        # Set defualt as Video
+        file_format = FileFormatType.VIDEO
+        other_options = None
```

Comment:
I will suggest just having a file_options dictionary with file_format as a key. We may have many possible options in future (e.g., the convert can be option). This saves us from changing the data structure of node and plan everytime when we add one option.

Reply:
Got it.